redwood-lazylayout-testing/src/commonMain/kotlin/app/cash/redwood/lazylayout/compose/DirectLoadingStrategy.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2024 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.lazylayout.compose
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+
+/**
+ * A loading strategy that's appropriate for tests because it's simple and predictable.
+ *
+ * This is not suitable for production use it must show a placeholder before an item is loaded.
+ */
+public class DirectLoadingStrategy : LoadingStrategy {
+  private var loadRange: IntRange by mutableStateOf(0..0)
```

Comment:
```suggestion
  private var loadRange: IntRange = 0..0
```
The only read is `firstIndex` which is only used when saving. Don't think you need a full-on state object tracking this value.

Reply:
Don’t I need this to trigger a recomposition after the user scrolls?

Reply:
I don't think the composition reads this. Maybe I missed it.

Reply:
It calls loadRange(), which reads variables here. I assume using State variables is the easiest way to express the data dependency.